// nextjs-tui-date-picker/dist/nextjs-tui-date-picker.cjs.prod.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var React = require('react');
var dynamic = require('next/dynamic');
var _extends = require('@babel/runtime/helpers/extends');

function _interopDefault (e) { return e && e.__esModule ? e : { 'default': e }; }

var React__default = /*#__PURE__*/_interopDefault(React);
var dynamic__default = /*#__PURE__*/_interopDefault(dynamic);

var NoSsrDatePicker = dynamic__default["default"](function () {
  return Promise.resolve().then(function () { return require('./date-picker-43c09c95.cjs.prod.js'); });
}, {
  ssr: false
});
var TuiDatePicker = function TuiDatePicker(props) {
  return /*#__PURE__*/React__default["default"].createElement(NoSsrDatePicker, props);
};
var TuiDatePicker$1 = TuiDatePicker;

var TuiDateTimePicker = function TuiDateTimePicker(props) {
  return /*#__PURE__*/React__default["default"].createElement(TuiDatePicker$1, _extends({}, props, {
    timePicker: true
  }));
};
var TuiDateTimePicker$1 = TuiDateTimePicker;

var TuiDateMonthPicker = function TuiDateMonthPicker(props) {
  return /*#__PURE__*/React__default["default"].createElement(TuiDatePicker$1, _extends({}, props, {
    dateType: "month"
  }));
};
var TuiDateMonthPicker$1 = TuiDateMonthPicker;

var TuiDateYearPicker = function TuiDateYearPicker(props) {
  return /*#__PURE__*/React__default["default"].createElement(TuiDatePicker$1, _extends({}, props, {
    dateType: "year"
  }));
};
var TuiDateYearPicker$1 = TuiDateYearPicker;

exports.TuiDateMonthPicker = TuiDateMonthPicker$1;
exports.TuiDatePicker = TuiDatePicker$1;
exports.TuiDateTimePicker = TuiDateTimePicker$1;
exports.TuiDateYearPicker = TuiDateYearPicker$1;
