// nextjs-tui-date-picker/dist/nextjs-tui-date-picker.esm.js
import React from 'react';
import dynamic from 'next/dynamic';
import _extends from '@babel/runtime/helpers/esm/extends';

var NoSsrDatePicker = dynamic(function () {
  return import('./date-picker-ea72acfe.esm.js');
}, {
  ssr: false
});
var TuiDatePicker = function TuiDatePicker(props) {
  return /*#__PURE__*/React.createElement(NoSsrDatePicker, props);
};
var TuiDatePicker$1 = TuiDatePicker;

var TuiDateTimePicker = function TuiDateTimePicker(props) {
  return /*#__PURE__*/React.createElement(TuiDatePicker$1, _extends({}, props, {
    timePicker: true
  }));
};
var TuiDateTimePicker$1 = TuiDateTimePicker;

var TuiDateMonthPicker = function TuiDateMonthPicker(props) {
  return /*#__PURE__*/React.createElement(TuiDatePicker$1, _extends({}, props, {
    dateType: "month"
  }));
};
var TuiDateMonthPicker$1 = TuiDateMonthPicker;

var TuiDateYearPicker = function TuiDateYearPicker(props) {
  return /*#__PURE__*/React.createElement(TuiDatePicker$1, _extends({}, props, {
    dateType: "year"
  }));
};
var TuiDateYearPicker$1 = TuiDateYearPicker;

export { TuiDateMonthPicker$1 as TuiDateMonthPicker, TuiDatePicker$1 as TuiDatePicker, TuiDateTimePicker$1 as TuiDateTimePicker, TuiDateYearPicker$1 as TuiDateYearPicker };
